pkgs/applications/networking/cluster/kns/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenvNoCC, lib, fetchFromGitHub, fzf, kubectl }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "kns";
+  version = "unstable-2022-04-25";
+
+  src = fetchFromGitHub {
+    owner = "blendle";
+    repo = pname;
```

Comment:
Please don't use this.

`repo = pname;` is nice for DRY but creates a binding that goes too far.

See further information about this here: https://github.com/nix-community/nixpkgs-lint/issues/21